internal/action/render.go

```diff
@@ -165,9 +166,55 @@ func sqliteToDeclcfg(ctx context.Context, dbFile string) (*declcfg.DeclarativeCo
 	}
 
 	cfg := declcfg.ConvertFromModel(m)
+
+	if err := populateDBRelatedImages(ctx, &cfg, db); err != nil {
+		return nil, err
+	}
 	return &cfg, nil
 }
 
+func populateDBRelatedImages(ctx context.Context, cfg *declcfg.DeclarativeConfig, db *sql.DB) error {
+	rows, err := db.QueryContext(ctx, "SELECT image, operatorbundle_name FROM related_image")
+	if err != nil {
+		return err
+	}
+	images := map[string]sets.String{}
+	for rows.Next() {
```

Comment:
I was blank on whether you need to defer the rows.Close() here. Looking at the godocs, it looks like it should be safe assuming rows.Next() eventually returns false, as the rows will get closed automatically.

Reply:
Good catch. Can't hurt to do it explicitly.

Reply:
Done!